pkg/kubectl/cmd/portforward/portforward_test.go

```diff
@@ -157,7 +157,7 @@ func TestTranslateServicePortToTargetPort(t *testing.T) {
 				Spec: corev1.ServiceSpec{
 					Ports: []corev1.ServicePort{
 						{
-							Port:       80,
```

Comment:
I think it's better to squash this commit into https://github.com/kubernetes/kubernetes/pull/78802.

Reply:
+1
please move these changes in https://github.com/kubernetes/kubernetes/pull/78802

thanks.
/close